### DISCUSSION TITLE
Incorporate an annotation tool into papis

### DISCUSSION THREAD

**Discussion by manuelhaussmann** [papis/papis#372] [Q&A] [answered]
I was wondering what the best way would be to combine papis with an annotation tool like [xournalpp](https://github.com/xournalpp/xournalpp). Essentially, I would like Papis to not only track the papers but also the corresponding annotation files (including the option to either open the pdf or the annotated version).

Do I see it correctly that the most straight forward way would be to dig into the [custom scripts](https://papis.readthedocs.io/en/latest/scripting.html) material? Or does anybody know whether there are already approaches/solutions out there among the papis users?

**Comment by alejandrogallo** [comment, marked as answer]
You can use the 
[notes-name](https://papis.readthedocs.io/en/latest/configuration.html#config-settings-notes-name)
configuration option and sete it to
`notes.xournalpp` or whatever the name of xournalpp files is.

Then do

```
papis edit --notes -e xournalpp
```

I think this might be enough for you, if I understood correctly.

**Reply by manuelhaussmann** [reply]
Thanks! I overlooked that option. That indeed solves my task completely.
